Week2/Pawfee/src/model/people/Employee.kt

```diff
@@ -0,0 +1,59 @@
+package model.people
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+private val dateFormatter = SimpleDateFormat.getTimeInstance()
+
+class Employee(
```

Comment:
Is there a specific reason you didn't create this as a `data class` instead of writing all of the method implementations manually?

Reply:
Can't have non val or var properties/parameters in a data class, and as we're using subclassing, we can't do it here, unless we explicitly override all of the properties, which doesn't make as much sense.